### PR TITLE
Eighth task

### PR DESCRIPTION
**Pull request by synion**
@kovson 
i hope this what i should do in services object

### CODE REVIEW

File: app/controllers/regard_mailer_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class RegardMailerController < ApplicationController
+
+  def create
+    current_user_id = params[:current_user_id]
```
Comment (kovson) [inline app/controllers/regard_mailer_controller.rb]: We should have access to current_user thanks to devise gem.

File: app/views/users/_user.html.haml [hunk 2]
```diff
@@ -12,9 +12,12 @@
       %td= user.age
       %td
         - if current_user.admin?
-          =link_to user_path(user), { method: :delete, data: { :confirm => 'Are you sure?' },  title:'delete' } do
+          = link_to user_path(user), { method: :delete, data: { :confirm => 'Are you sure?' },  title:'delete' } do
             %button.btn.btn-default.index-button
               = fa_icon "trash"
+        = link_to send_regard_index_path(user_id: user, current_user_id: current_user ), { method: :post } do
```
Comment (kovson) [inline app/views/users/_user.html.haml]: We should have access to the current user in a controller, so probably sending current_user is not necessary here.

File: app/views/regard_mailer/send_regard.text.haml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+= "#{ @current_user_email } send his regards"
+
```
Comment (kovson) [inline app/views/regard_mailer/send_regard.text.haml]: ✂️

File: app/views/regard_mailer/send_regard.html.haml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+%p= "#{ @current_user_email } send regards"
+
```
Comment (kovson) [inline app/views/regard_mailer/send_regard.html.haml]: ✂️

File: spec/controllers/regard_mailer_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe RegardMailerController, type: :controller do
```
Comment (kovson) [inline spec/controllers/regard_mailer_controller_spec.rb:3]: It will be changed.

File: spec/services/send_regard_spec.rb [hunk 6]
```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe SendRegard do
```
Comment (kovson) [inline spec/services/send_regard_spec.rb:3]: 👍  Nice.

File: app/mailers/regard_mailer.rb [hunk 7]
```diff
@@ -0,0 +1,8 @@
+class RegardMailer < ApplicationMailer
+
+  def send_regard(options)
+    @user_email = options[:user_email]
+    @current_user_email = options[:current_user_email]
+    mail  to: @user_email
```
Comment (kovson) [inline app/mailers/regard_mailer.rb]: extra space ;>

File: config/environments/development.rb [hunk 8]
```diff
@@ -25,6 +25,9 @@
 
     config.cache_store = :null_store
   end
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = { :address => '127.0.0.1', :port => 1025 }
+
```
Comment (kovson) [inline config/environments/development.rb]: ✂️

**Comment by kovson** [comment]
@synion 
Very few small changes, impressive job. Please squash or amend commits with feedback.

File: app/views/users/_user.html.haml (hunk 2 above)
Comment (synion) [reply app/views/users/_user.html.haml]: @kovson 
omg you have right why i im send this i dont need this :)

**Comment by synion** [comment]
@kovson 
I try amend and squash to

**Pull request merged**